CONTRIBUTING.md

```diff
@@ -128,6 +128,20 @@ make server
 Any time changes are made to the Go code, you will have to stop the server and run the above command again.
 Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`.
 
+**Please note**: If you get error while starting the server as **"Meshery Development Incompatible"** then follow the below guideline 👇
+
+<img src="./docs/assets/img/meshery-development-incompatible-error.png" width="50%">
```

Comment:
This reference is invalid.

Reply:
Sorry I didn't get it what you are trying to point out with "this reference" 
Is it the image which is referencing to invalid? or text written above? 
Can you please correct me? So I can work on your requested changes to the soonest and we can merge this PR if rest looks good. 

Cheers.

Reply:
Or are you telling the image path we gave is invalid? Shall I add image in the global directory and reference that path to this `src` ? Please resolve my doubt, thanks 😄 🤔 @leecalcote

Reply:
It's possible that I'm wrong, let's see...